### ISSUE TITLE
Problems installing plugin to UE 5

### ISSUE THREAD

**Issue by HanSchne** [labstreaminglayer/plugin-UE4#9]
I am having issues installing the LSL plugin into my Unreal Engine 5 (version 5.0.2 as of now).
I have tried both the installation over the Epic Games Store as well as copying the plugin from this repository into my Project folder (/[Project Root]/Plugins) as per instructions.

When attempting to install over the Epic Games Store, after selecting the engine version "5.0" and clicking "Install" nothing happens and I receive no feedback. The same can not be said after attempting to install it to UE 4.27, which went as expected (where I was also able to use the plugin as intended).

When attempting to install by copying the folder from GitHub into my project folder, I receive a command prompt when launching my project asking me this:
"The following modules are missing or built with a different engine version:
LSL
Would you like to rebuild them now?
[Yes] [No]"
After clicking Yes I immediately receive an error prompt stating: "{Project Name} could not be compiled. Try rebuilding from source manually."
I am able to launch the project again after removing the folder.

Are you aware of the issue and know what causes it?
I am really happy so far with what the plugin does in UE 4.27 and I would love to be able to use it in the powerful UE 5.

**Comment by brifsttar** [comment]
Hi, thank you for your message and detailed explanation.

For the first issue, I've tried installing the plugin from the Launcher, and indeed, nothing happens. That's really weird, I went through the Launcher logs, and the issue seems to be on their side. There were quite a few hiccups with regards to Marketplace plugins at the initial 5.0 release, but they should have been fixed by now. Anyway, I've contacted Epic, I'll let you know once it's fixed.

For the second issue, I wasn't able to reproduce it. I created a blank UE5 project and cloned this plugin into `Plugins/`. I was also prompted to rebuild the modules, but it was successful. You can check (and/or share) the log file (in `Saved\Logs\`) so that we can get a better understanding of why the rebuilding failed. Do you have the Unreal C++ compilation toolchain installed (e.g., VS2019, .NET SDK 4.6)? If not, that's probably the issue.

I can help you rebuild from source, or if you don't want to set that up, I'll let you know as soon as the Launcher issue is fixed.

**Comment by brifsttar** [comment]
Epic answered

> I made an adjustment on our end. Please fully restart your Launcher and then try again, and let us know if you are still having issues. 

It now works on my end, let me know if it's ok for you.

**Comment by HanSchne** [comment]
Thank you for the quick reply!

First of: The Epic Games Store Marketplace installation works now as expected for me as well.

Secondly: After checking the log files, it turned out that I was indeed missing the .NET SDK. I did not expect that as I had VS22 installed with the C++ game development package. Installing the SDK did not fix it completely. I checked the Log files again and this time I had an incompatible framework: The package contains .NET framework 6.1, but LSL requires 3.1 (correct me here, if I am wrong).
This was also easily fixed by simply installing it in the installer and after that I was able to install your plugin the manual way as well!
So in conclusion if you are using VS22:
1. Install C++ game development package
2. Install .NET SDK
3. Install .NET framework 3.1 LTS
This should suffice to be able to install the plugin manually in my experience.

I will use the Marketplace installation method out of convenience in the future.
I hope I was as helpful to you as you were to me.
